docs/source/dask.rst

```diff
@@ -0,0 +1,98 @@
+and manually starting a dask cluster with UCX enabled.  Additionally, we demonstrate using UCX with a `cuDF Example`_ and `CuPy Example`_
+
+Starting with Dask-cuda
+-----------------------
+
+`dask-cuda <https://github.com/rapidsai/dask-cuda>`_ is Python tool designed to help with cluster deployment and management of Dask workers on CUDA-enabled systems.  Dask-cuda can be used to in two ways: inline and CLI.
+
+Inline
+~~~~~~
+
+.. code-block:: python
+
+    from dask.distributed import Client
+    from dask_cuda import DGX
+    from dask_cuda.initialize import initialize
+
+    # ON/OFF settings for various devices
+    enable_tcp_over_ucx = True
+    enable_infiniband = False
+    enable_nvlink = False
+
+
+    # initialize client with the same settings as workers
+    initialize(
+        create_cuda_context=True,
+        enable_tcp_over_ucx=enable_tcp_over_ucx,
+        enable_infiniband=enable_infiniband,
+        enable_nvlink=enable_nvlink,
+    )
+
+    cluster = DGX(
+        interface="enp1s0f0",
+        protocol="ucx",
+        enable_tcp_over_ucx=enable_tcp_over_ucx,
+        enable_infiniband=enable_infiniband,
+        enable_nvlink=enable_nvlink,
+    )
+    client = Cluster(client)
+
+CLI
+~~~~
+
+Dask-cuda can also be used when manually starting a cluster:
+
+.. code-block:: bash
+
+    # server
+    UCX_CUDA_IPC_CACHE=n UCX_MEMTYPE_CACHE=n UCX_TLS=tcp,sockcm,cuda_copy,cuda_ipc \
+    UCX_SOCKADDR_TLS_PRIORITY=sockcm python -m distributed.cli.dask_scheduler --interface enp1s0f0 --protocol ucx
+
+    # worker
+    UCX_CUDA_IPC_CACHE=n UCX_TLS=tcp,sockcm,cuda_copy,cuda_ipc \
+    UCX_SOCKADDR_TLS_PRIORITY=sockcm dask-cuda-worker ucx://{SCHEDULER_ADDR}:8786
+
+    # client
+    UCX_CUDA_IPC_CACHE=n UCX_TLS=tcp,cuda_copy,cuda_ipc,sockcm \
+    UCX_SOCKADDR_TLS_PRIORITY=sockcm python <python file>
+
+
+The benefit of using ``dask-cuda-worker`` is that it will invoke N workers where N is the number of GPUs and automatically pair workers with GPUs.
+
+
+Manual Cluster Creation
+-----------------------
+
+Lastly, we can also manually start each worker individually (this is typically only used when debugging):
+
+.. code-block:: bash
+
+    # server
+    UCX_CUDA_IPC_CACHE=n UCX_MEMTYPE_CACHE=n UCX_TLS=tcp,sockcm,cuda_copy,cuda_ipc \
+    UCX_SOCKADDR_TLS_PRIORITY=sockcm python -m distributed.cli.dask_scheduler --interface enp1s0f0 --protocol ucx
+
+    # worker
+    CUDA_VISIBLE_DEVICES=0 UCX_CUDA_IPC_CACHE=n UCX_TLS=tcp,sockcm,cuda_copy,cuda_ipc \
+    UCX_SOCKADDR_TLS_PRIORITY=sockcm dask-worker ucx://{SCHEDULER_ADDR}:8786
+
+    # client
+    UCX_CUDA_IPC_CACHE=n UCX_TLS=tcp,cuda_copy,cuda_ipc,sockcm \
+    UCX_SOCKADDR_TLS_PRIORITY=sockcm python <python file>
+
+Note: ``CUDA_VISIBLE_DEVICES`` controls which GPU(s) the worker has access to.
```

Comment:
I would say add another one here, but maybe we should have a single note about the ethernet interface and explain that it should be used for the cases of all examples.